controllers/dspipeline_controller.go

```diff
@@ -161,7 +163,7 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	err := r.Get(ctx, req.NamespacedName, dspa)
 	if err != nil && apierrs.IsNotFound(err) {
 		log.Info("Stop DSPAParams reconciliation")
-		return ctrl.Result{}, nil
+		return ctrl.Result{Requeue: true, RequeueAfter: 2 * time.Minute}, nil
```

Comment:
this is in the wrong place, the logs are misleading, probably became outdated since the params extraction was moved further down, the logs should just read: 

> "Stop DSPAParams reconciliation"  ---> "DSPA resources was not found."
> "Unable to fetch the DSPAParams" --> "Encountered error when fetching DSPA."

As for where we should requeue, it's here: https://github.com/opendatahub-io/data-science-pipelines-operator/blob/dc86c494ecd551a1aea10d426c8951068d8e8adb/controllers/dspipeline_controller.go#L209

Reply:
Irrespective of the misleading logs, I assumed based on our previous discussion, this was the right place to place the periodic reconciliation even though ExtractParams made more sense. I made the changes now and changed the log messages as suggested which makes more sense.